services/data-collection/native-cookie-support.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Native Cookie Support
+description: Manage analytics cookies directly at the edge for unified user tracking.
+icon: cookie
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Native Cookie Support is a powerful feature that allows Edgee to manage analytics cookies directly at the edge, 
+allowing you to maintain consistent user identification between Edgee and the analytics platform's JS SDK.
+
+This feature is particularly useful when you're using both the Google Analytics client-side JavaScript SDK and Edgee's Google Analytics component simultaneously, as it ensures consistent user tracking across both implementations.
```

Comment:
I'd suggest generalizing here (instead of focusing on Google Analytics specifically)